_config.yml

```diff
@@ -1,4 +1,5 @@
 # Build settings
-markdown: kramdown
+kramdown:
```

Comment:
This doesn't work somehow, even though it should as described [here](https://jekyllrb.com/docs/configuration/markdown/). Someone has an idea?

Reply:
I realized that this obviously cannot work as we don't use Kramdown, but Haml...